pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -818,7 +856,7 @@ func decodeSnapshotID(snapshotID astrolabe.ProtectedEntitySnapshotID, logger log
 		logger.WithError(err).Error(errorMsg)
 		return "", errors.Wrap(err, errorMsg)
 	}
-	logger.Infof("Successfully translated snapshotID %s into pe-id: %s", string(snapshotIDBytes), decodedPEID.String())
+	logger.Infof("Successfully translated snapshotID %s into pe-id: %s", snapshotID.String(), decodedPEID.String())
 	if decodedPEID.HasSnapshot() && decodedPEID.GetPeType() != "ivd" {
 		logger.Infof("The translated pe-id is not ivd type, recurring for further decode")
```

Comment:
This was already there - logger.Infof("The translated pe-id is not ivd type, recurring for further decode") but should be "recursing" not "recurring"

Reply:
Changed to recursing, the previous log message printed out the decoded string(snapshotIDBytes), I actually intended to print the incoming encoded snapshotID.